read-cache.c

```diff
@@ -2229,7 +2229,7 @@ static int should_delete_shared_index(const char *shared_index_path)
 	if (!expiration)
 		return 0;
 	if (stat(shared_index_path, &st))
-		return error_errno(_("could not stat '%s"), shared_index_path);
+		return error_errno(_("could not stat '%s'"), shared_index_path);
```

Comment:
Ah, here is the other error:

https://github.com/git-l10n/git-po/blame/master/builtin/tag.c#L313

I don't know how to sprintf with strbuf